lib/plotTimesFreqfig.m

```diff
@@ -1,12 +1,12 @@
-function [] = plotTimesFreqfig(T,F)
-
-%PLOTALL Summary of this function goes here
+function [] = plotTimesFreqfig(T,F,friv)
```

Comment:
I think that this function can be cancelled, and the "new*" re-named to this one